core-bundle/src/Migration/CommandCompiler.php

```diff
@@ -28,13 +28,45 @@ public function __construct(private readonly Connection $connection, private rea
     /**
      * @return list<string>
      */
-    public function compileCommands(): array
+    public function compileCommands(bool $doNotDropColumns = false): array
```

Comment:
```suggestion
    public function compileCommands(bool $skipDropStatements = false): array
```
We should also not drop tables I guess?

Reply:
True that :smile:

Reply:
See 66652ccc9f931151f18ed1d0e0d6a66d23982f6d + d22dbbfb2b3883b9c41d9fe39e17633dfc8ca38e.

I also went with reflection here, because this way copying definitions is much saner. Unfortunately.

Reply:
I looked into this and reflection seems to be the best way to do this 😕